### PR TITLE
gh-113791: Expose CLOCK_MONOTONIC_RAW_APPROX and CLOCK_UPTIME_RAW_APROX on macOS in the time module

### PR DESCRIPTION
**Pull request by ronaldoussoren**
* Issue: gh-113791

----
📚 Documentation preview 📚: https://cpython-previews--113792.org.readthedocs.build/

### CODE REVIEW

**Comment by ronaldoussoren** [comment]
There are no test updates because existing CLOCK_* constants aren't tested either (other than indirectly testing some of them through tests of functions in the module)

File: Modules/timemodule.c [hunk 1]
```diff
@@ -2048,6 +2048,18 @@ time_exec(PyObject *module)
         if (PyModule_AddIntMacro(module, CLOCK_UPTIME_RAW) < 0) {
             return -1;
         }
+#endif
+#ifdef CLOCK_MONOTONIC_RAW_APPROX
+
```
Comment (corona10) [inline Modules/timemodule.c]:
```suggestion
```

File: Modules/timemodule.c [hunk 2]
```diff
@@ -2048,6 +2048,18 @@ time_exec(PyObject *module)
...
+        if (PyModule_AddIntMacro(module, CLOCK_MONOTONIC_RAW_APPROX) < 0) {
+            return -1;
+        }
+#endif
+#ifdef CLOCK_UPTIME_RAW_APPROX
+
```
Comment (corona10) [inline Modules/timemodule.c]:
```suggestion
```

**Review by corona10 — COMMENTED** [review]
LGTM, But let's follow convention of other constants :)

**Comment by corona10** [comment]
> LGTM, But let's follow convention of other constants :)

Ah, some of the other constants have empty space. Can we unify them through this PR?

**Review by erlend-aasland — APPROVED** [review]

**Comment by ronaldoussoren** [comment]
> > LGTM, But let's follow convention of other constants :)
> 
> Ah, some of the other constants have empty space. Can we unify them through this PR?

That's why my entries have empty lines, I copied the last entry in the list :-) . I'll remove the blank lines.

**Review by erlend-aasland — APPROVED** [review]

**Review by corona10 — APPROVED** [review]

**Pull request merged**